modules/serverless-customize-labels-annotations-routes.adoc

```diff
@@ -0,0 +1,68 @@
+[id="serverless-customize-labels-annotations-routes_{context}"]
+= Customizing labels and annotations for {product-title} routes
+
+{product-title} routes support the use of custom labels and annotations, which can be configured by modifying the `metadata` spec of a Knative service. Custom labels and annotations are propagated from the service to the Knative route, then to the Knative ingress, and finally to the {product-title} route.
+
+The following annotations are supported by Knative Serving:
+
+* Disabling routes, which is specified by the `serving.knative.openshift.io/disableRoute` annotation.
+* Enabling pass-through routes, which is specified by the `serving.knative.openshift.io/enablePassthrough` annotation.
+
+.Prerequisites
+
+* You must have the {ServerlessOperatorName} and Knative Serving installed on your {product-title} cluster.
+
+.Procedure
+
+. Create a Knative service that contains the label or annotation that you want to propagate to the {product-title} route:
++
+.Example service YAML
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: <service_name>
+  labels:
+    <label_name>: <label_value>
+  annotations:
+    <annotation_name>: <annotation_value>
+...
+----
++
+.Example of enabling pass-through routes by using the annotation
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: example-service
+...
+  annotations:
+    "serving.knative.openshift.io/enablePassthrough": "true"
+...
+----
```

Comment:
Please use example of generic labels and annotations, like it was used in draft. Using this special annotation is misleading.

```suggestion
.Example of passing custom labels and annotations to route
[source,yaml]
----
apiVersion: serving.knative.dev/v1
kind: Service
metadata:
  name: example
  labels:
    staging: true
  annotations:
    my-annotation: some-value
...
----
```

Or just remove this example and stay with generic `<label_name>:<label_value>`, `<annotation_name>: <annotation_value>`. That's also okay.